.gitignore

```diff
@@ -15,3 +15,6 @@ node_modules
 
 .eslintcache
 .pgdata
+.build
+yarn.build.json
+packing.sh
```

Comment:
what is this shell script we're ignoring?

Reply:
it is WIP